test/e2e/jest-playwright.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
     usedPortAction: 'ignore', // your test are executed, we assume that the server is already started
   },
   launchOptions: {
-    headless: process.env.HEADLESS !== 'false',
+    headless: process.env.HEADLESS !== false,
```

Comment:
⚠️ not related to this PR. 
❓ why is this needed, I was previously able to enable/disable HEADLESS without this change. Did you have any issue with the existing code? `process.env.HEADLESS` is supposed to return a `string` so the condition seemed valid as well.

Reply:
I used it to debug the problem that we have on webkit. And I found another problem with a bad type for the value of this property.